po/tr.po

```diff
@@ -50,273 +50,67 @@ msgid ""
-msgid "More fun for everyone"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:117
-msgid "Added eight more original replies"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:118
-msgid "Added “About” window"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:119
-msgid ""
-"Improvements to developer tools to make translations easier to keep up to "
-"date"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:120
-msgid "Italian translations thanks to Albano Battistella"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:121
-msgid "Updated runtime to GNOME 43"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:127
-msgid ""
-"Hotfix: fix a crash on startup from trying to access nonexistent settings"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:132
-msgid "Hello, GNOME"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:134
-msgid "Whole new UI"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:135
-msgid "elementary → GNOME styling"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:136
-msgid ""
-"Technical upgrade to the latest GNOME platform including GTK4 and LibAdwaita"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:142
-msgid "Hello, Odin"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:144
-msgid "Updated for OS 6 and Flatpak"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:145
-msgid "Turkish translations thanks to Safak GENISOL"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:146
-msgid "Portuguese translations thanks to André Barata"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:147
-msgid "Spanish translations thanks to @kevincos5"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:153
-msgid "Refreshed icons thanks to Micah Ilbery"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:158
-msgid "Polish translations thanks to Michał Nowakowski"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:163
-msgid "Dutch translations thanks to Heimen Stoffels"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:168
-msgid "French translations thanks to @NathanBnm"
-msgstr ""
-
-#: data/metainfo.appdata.xml.in:173
-msgid "Clairvoyant is now translatable."
-msgstr "Clairvoyant artık tercüme edilebilir."
-
-#: data/metainfo.appdata.xml.in:178
-msgid "Happy new year! This release contains AppData fixes."
-msgstr "Yeni Yılınız mutlu olsun! Bu sürüm, AppData düzeltmelerini içerir."
-
-#: data/metainfo.appdata.xml.in:183
-msgid "Fix 48px icon"
-msgstr "48px simgesini düzeltildi"
-
-#: data/metainfo.appdata.xml.in:188
-msgid "Post-release update"
-msgstr "Yayın sonrası güncelleme"
-
-#: data/metainfo.appdata.xml.in:190
-msgid "Clean up AppStream data"
-msgstr "AppStream verilerini temizleyin"
-
-#: data/metainfo.appdata.xml.in:191
-msgid "Expand on the description"
-msgstr "Açıklamayı genişletin"
-
-#: data/metainfo.appdata.xml.in:192
-msgid "Change price to $1"
-msgstr "Fiyatı 1 $ olarak değiştirin"
-
-#: data/metainfo.appdata.xml.in:198 src/FortuneLabel.vala:20
-msgid "It is decidedly so."
-msgstr "Kesinlikle öyle."
-
-#: data/metainfo.appdata.xml.in:200
-msgid "Initial release"
-msgstr "İlk sürüm"
```

Comment:
Is there a reason these have been removed?

Reply:
I'll put them back. I assumed the other MR was accepted.

https://github.com/cassidyjames/clairvoyant/pull/64